institutions/views.py

```diff
@@ -0,0 +1,17 @@
+"""Views for the institutions app."""
+
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from institutions.models import Institution
+from institutions.serializers import InstitutionSerializer
+
+
+class InstitutionViewSet(viewsets.ModelViewSet):
+    """ViewSet for the Institution model."""
+    queryset = Institution.objects.all().prefetch_related('domains')
+    serializer_class = InstitutionSerializer
+
+    def list(self, request, **kwargs):
+        serializer = InstitutionSerializer(self.queryset, many=True)
+        return Response(serializer.data)
```

Comment:
ModelViewSet provides all the standard actions, including list(). You don't need to override it since you're not doing any custom logic.

Reply:
Indeed. I just wrote this here to remind me how it worked.